bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -1,321 +1,835 @@
+        },
+        location: {
+          displayName: actionParameters.location,
+        },
+        attendees: [
+          {
+            emailAddress: {
+              address: actionParameters.email1,
+            },
+            type: actionParameters.type1,
+          },
+          {
+            emailAddress: {
+              address: actionParameters.emai2,
+            },
+            type: actionParameters.type2,
+          },
+          {
+            emailAddress: {
+              address: actionParameters.email3,
+            },
+            type: actionParameters.type3,
+          },
+          {
+            emailAddress: {
+              address: actionParameters.email4,
+            },
+            type: actionParameters.type4,
+          },
+          {
+            emailAddress: {
+              address: actionParameters.email5,
+            },
+            type: actionParameters.type5,
+          },
+          {
+            emailAddress: {
+              address: actionParameters.email6,
+            },
+            type: actionParameters.type6,
+          },
+        ],
+        allowNewTimeProposals: true,
+        isOnlineMeeting: actionParameters.isOnlineMeeting,
+        onlineMeetingProvider: actionParameters.onlineMeeting,
+        id: actionParameters.id,
+        //userId: actionParameters.userId
+      }),
+    }
+  );
+  console.log(JSON.stringify(responseOfCurrentTimeZone));
+  if (!responseOfCurrentTimeZone.ok) {
+    throw new Error(
+      `Could not Edit Event (${responseOfCurrentTimeZone.status}: ${responseOfCurrentTimeZone.statusText})`
+    );
+  }
+  const user_Id = [actionParameters.userId];
+  await calendarView(client, dataStore, user_Id, startDate);
+  await myEvents(client, dataStore, user_Id);
 }
+//EditOneTimeEventWithCustomTimezone
+async function EditOneTimeEventWithCustomTimezone({
+  dataStore,
+  client,
+  actionParameters,
+}) {
+  const startDate = moment.utc().subtract(1, "m").format();
+
+  const responseOfCoustomTimeZone = await client.fetch(
+    `v1.0/me/events/${actionParameters.id}`,
+    {
+      method: "PATCH",
+      body: JSON.stringify({
+        subject: actionParameters.subject,
+        body: {
+          contentType: "HTML",
+          content: actionParameters.content,
+        },
+        start: {
+          dateTime: `${actionParameters.startDate}T${actionParameters.startTime}`,
+          timeZone: actionParameters.timezone,
+        },
+
+        end: {
+          dateTime: `${actionParameters.endDate}T${actionParameters.endTime}`,
+
+          timeZone: actionParameters.timezone,
+        },
+        location: {
+          displayName: actionParameters.displayName,
+        },
+        attendees: [
+          {
+            emailAddress: {
+              address: actionParameters.email1,
+            },
+            type: actionParameters.type1,
+          },
+          {
+            emailAddress: {
+              address: actionParameters.emai2,
```

Comment:
Here it is `address: actionParameters.emai2` but in the action parameters schema it is `email2`. Name of the variable is mismatched.
Is this script really tested? If yes, then this should be results in error.